src/Protocol/Protocol_1_13.h

```diff
@@ -68,10 +69,14 @@ class cProtocol_1_13 :
 	virtual void SendPaintingSpawn              (const cPainting & a_Painting) override;
 	virtual void SendPluginMessage              (const AString & a_Channel, const AString & a_Message) override;
 	virtual void SendScoreboardObjective        (const AString & a_Name, const AString & a_DisplayName, Byte a_Mode) override;
+	virtual void SendSpawnMob                   (const cMonster & a_Mob) override;
 	virtual void SendStatistics                 (const cStatManager & a_Manager) override;
 	virtual void SendTabCompletionResults       (const AStringVector & a_Results) override;
 	virtual void SendUpdateBlockEntity          (cBlockEntity & a_BlockEntity) override;
 
+	/** Converts eMonsterType to protocol-specific mob IDs */
+	static UInt32 GetProtocolMobID(eMonsterType a_MobType);
+
```

Comment:
This should not be `static` at all, but rather `virtual` - pure virtual in `cProtocol` and `virtual override` in all descendants.

Reply:
Make it return `UInt32` in all versions, then `static_cast` in protocols that use shorter datatype.